src/components/attendee/InputSelect/InputSelect.tsx

```diff
@@ -18,70 +18,83 @@ interface InputSelectProps
     ContainerClassName {
   inputClassName?: string;
   errorTextClassName?: string;
-  error?: FieldError;
+  errorSelect?: FieldError;
+  errorInput?: FieldError;
   inputName: string;
   selectName: string;
   inputPlaceholder?: string;
   selectPlaceholder?: string;
   register: UseFormRegister<AnyForm>;
-  inputRules: RegisterOptions;
-  selectRules: RegisterOptions;
+  inputRules?: RegisterOptions;
+  selectRules?: RegisterOptions;
 }
 
 export const InputSelect: React.FC<InputSelectProps> = ({
   containerClassName,
   inputClassName,
   errorTextClassName,
-  error,
+  errorSelect,
+  errorInput,
   inputName,
   selectName,
   inputPlaceholder,
   selectPlaceholder,
   register,
-  inputRules,
-  selectRules,
+  inputRules = {},
+  selectRules = {},
 }) => {
   const containerClassNames = classNames(
     styles.inputSelect,
     containerClassName
   );
 
   const wrapperClassNames = classNames(styles.inputWrapper, {
-    [styles.invalid]: error,
+    [styles.invalid]: errorInput,
   });
 
   const inputClassNames = classNames(styles.input, inputClassName);
+
+  const selectWrapperClassNames = classNames(styles.selectWrapper, {
+    [styles.invalid]: errorSelect,
+  });
   const selectClassNames = classNames(styles.select, containerClassNames);
 
   return (
     <div className={containerClassNames}>
-      <div className={styles.wrapper}>
+      <form className={styles.wrapper} autoComplete="off">
         <div className={wrapperClassNames}>
           <input
             className={inputClassNames}
             placeholder={inputPlaceholder}
+            autoComplete="off"
```

Comment:
do we need to state `autocomplete=off` in both places? I think input should be enough

Reply:
for some reason autocomplete disables only if we put it in the form and the inputs inside it, otherwise it still works